msg/ekf2_replay.msg

```diff
@@ -37,3 +37,10 @@ float32[2] flow_pixel_integral		# integrated optical flow rate around x and y ax
 float32[2] flow_gyro_integral		# integrated gyro rate around x and y axes (rad)
 uint32 flow_time_integral		# integration timespan (usec)
 uint8 flow_quality			# Quality of accumulated optical flow data (0 - 255)
+
+# vehicle status
+uint8 vehicle_status_mask		# mask of vehcle status - bit positions defined as:
+# 0 - true if the vehicle is landed
+# 1 - true if the landed status has been updated on this frame
+# 2 - true if the actuators/motors are armed
+# 3 - true if the armed status has been updated on this frame
```

Comment:
I will pull the logging patches out of the accuracy improvement PR and rework them them as a separate PR